lib/liquid_voting_web/schema/schema.ex

```diff
@@ -19,9 +20,9 @@ defmodule LiquidVotingWeb.Schema.Schema do
       resolve(&Resolvers.Voting.participants/3)
     end
 
-    @desc "Get a participant by its id"
+    @desc "Get a participant by its uuid"
     field :participant, :participant do
-      arg(:id, non_null(:id))
+      arg(:uuid, non_null(:string))
```

Comment:
wouldn't it be `non_null(:uuid)` now that the UUID custom type's been imported?

Reply:
for example, this custom `:datetime` type is used at the query arg level too: https://hexdocs.pm/absinthe/custom-scalars.html

```elixir
field :post, :post do
  arg :published_at, :datetime
```

Reply:
this looks like an oversight. I'll try it and test.